test/boulder-config.json

```diff
@@ -88,7 +88,7 @@
                 } ]
               }
             ],
-            "expiry": "8760h",
+            "expiry": "2160h",
```

Comment:
Someone never changed the expiration period for CFSSL to 90 days in the test config, so it has been issuing year long certificates. Bad not-prod config!